fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/OnlineIndexer.java

```diff
@@ -2585,14 +2593,21 @@ public Builder setMutualIndexing() {
              * @return this builder
              */
             @API(API.Status.EXPERIMENTAL)
-            public Builder setMutualIndexing(final boolean useMutualIndexing) {
+            public Builder setMutualIndexing(boolean useMutualIndexing) {
```

Comment:
I feel like it might be worth removing all the overrides of `setMutualIndexing`, and just have the one that takes to booleans. Given that they're all `EXPERIMENTAL`, we don't have to worry too much about backwards compatibility, and I think these overrides make it more confusing.

Reply:
✅ 
I've left `setMutualIndexing()` and compatibility to other features. This can be useful in tests and quick coding experiments.